packages/react-native-client/src/hooks/useCamera.ts

```diff
@@ -254,17 +220,58 @@ export function useCamera() {
   );
 
   return {
+    /**
+     * Informs if user camera is streaming video
+     */
     isCameraOn,
+    /**
+     * Which camera is now used for streaming (or will be used as default when camera will be enabled)
+     */
     currentCamera,
+    /**
+     * Simulcast configuration
+     */
     simulcastConfig,
+    /**
+     * Property that lists cameras available on device.
+     * @returns A promise that resolves to the list of available cameras.
+     */
     cameras,
 
+    /** Enable/disable current camera */
     toggleCamera,
-    prepareCamera,
-    switchCamera,
+    /**
+     * Prepares camera and starts local camera capture.
+     * @param config configuration of the camera capture
+     * @returns A promise that resolves when camera is started.
+     */ prepareCamera,
```

Comment:
**suggestion:** Maybe let's also mention the fact that it creates a local video track.